Tests/IndexStoreDBTests/IndexTests.swift

```diff
@@ -253,6 +253,73 @@ final class IndexTests: XCTestCase {
     XCTAssertEqual(delegate.completed, 3)
   }
 
+  func testOutOfDateEvent() throws {
```

Comment:
Need to update Linux tests.  I mentioned this to Alex for SourceKit-LSP, but we should consider switching to rely on automatic test discovery on Linux now that it's been around for a while.  We may need to bump our swift tools version in the package for that, not sure.

Reply:
Good catch! We missed updating for other tests as well. Done.